.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Install '....'
+3. Run commands '....'
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
```

Comment:
1. I think maybe "Environment" is more suitable thank "Desktop".
2. Also need to include the hardware information, especially GPU cards type.
3. And add below software information:
    (1) Python version.
    (2) PyTorch version.
    (3) Ignite version.
    (4) CUDA version(optional).
    (5) cuDNN version(optional).

Reply:
Thanks, I've updated the section. hope it's fine to keep it light-weight for now. in the future we should use a CLI script to automatically collect/show system info.